packages/twenty-front/src/modules/workspace-invitation/hooks/useInviteUser.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  useSendInviteLinkMutation,
+  SendInviteLinkMutationVariables,
+} from '~/generated/graphql';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+import { workspaceInvitationsState } from '../states/workspaceInvitationsStates';
+
+export const useInviteUser = () => {
+  const [sendInviteLinkMutation] = useSendInviteLinkMutation();
+
+  const setWorkspaceInvitations = useSetRecoilState(workspaceInvitationsState);
```

Comment:
you can do const [workspaceInvitations, setWorkspaceInvitations] = useRecoilState